bascule/metrics.go

```diff
@@ -0,0 +1,36 @@
+package bascule
+
+import (
+	"github.com/Comcast/webpa-common/xmetrics"
```

Comment:
bascule can't depend on `webpa-common`.  eventually, `webpa-common` needs to depend on `bascule`.